src/nvim/generators/gen_normal_cmds.lua

```diff
@@ -0,0 +1,48 @@
+local autodir = arg[1]
+
+local deffname = autodir .. '/normal_cmds.generated.h'
+local defsfile = io.open(deffname, 'w')
+
+defsfile:write [[
+#include "nvim/normal_defs.h"
+
+/// Function to be called for a Normal or Visual mode command.
+/// The argument is a cmdarg_T.
+typedef void (*nv_func_T)(cmdarg_T *cap);
+
+// Generally speaking, every Normal mode command should either clear any
+// pending operator (with *clearop*()), or set the motion type variable
+// oap->motion_type.
+//
+// When a cursor motion command is made, it is marked as being a character or
+// line oriented motion.  Then, if an operator is in effect, the operation
+// becomes character or line oriented accordingly.
```

Comment:
I don't think this block should be in a generated file.

Reply:
okay .